test/integration/options.spec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var path = require('path');
+var loadOptions = require('../../lib/cli/options').loadOptions;
+
+describe('options', function () {
+  var workingDirectory;
+  const workspaceDir = path.join(
+    __dirname,
+    'fixtures',
+    'config',
+    'mocharc-extended'
+  );
+  const configFile = path.join(workspaceDir, 'extends.json');
+
+  beforeEach(function () {
+    workingDirectory = process.cwd();
+    process.chdir(workspaceDir);
+  });
+
+  afterEach(function () {
+    process.chdir(workingDirectory);
+  });
+
+  it('Should load `_`-option properly', function () {
+    var extended = loadOptions(['--config', configFile]);
+    expect(extended._, 'to equal', ['**/*.spec.js']);
+  });
+
+  it('Should support extended options using --config parameter', function () {
+    var extended = loadOptions(['--config', configFile]);
+    expect(extended.require, 'to equal', ['foo', 'bar']);
+    expect(extended.bail, 'to equal', true);
+    expect(extended.reporter, 'to equal', 'html');
+    expect(extended.slow, 'to equal', 30);
+  });
+
+  it('Should support extended options using rc file', function () {
+    var extended = loadOptions([]);
+    expect(extended.require, 'to equal', ['foo', 'bar']);
+    expect(extended.bail, 'to equal', true);
+    expect(extended.reporter, 'to equal', 'html');
+    expect(extended.slow, 'to equal', 30);
+  });
+
```

Comment:
[Testing] When I first looked at this file I was confused that nothing ever tested the original values from `base.json`. I manually added this test case locally to confirm things still work. Any reason not to including it?

```suggestion

  it('Should support standalone options using standalone rc file', function () {
    var extended = loadOptions(['--config', configFileStandalone]);
    expect(extended.require, 'to equal', ['foo', 'bar']);
    expect(extended.bail, 'to equal', true);
    expect(extended.reporter, 'to equal', 'dot');
    expect(extended.slow, 'to equal', 60);
  });
```

...with above:

```js
const configFileStandalone = path.join(workspaceDir, 'base.json');
```